<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tianjiafriend</class>
 <widget class="QWidget" name="tianjiafriend">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tianjiafriend</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入账号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shuru_zhanghao">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chaxunbt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="chaxun_touxiang">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>性    别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="chaxun_xingbie">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账    号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="chaxun_zhanghao">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵    称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="chaxun_nicheng">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>个性签名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="chaxun_qianming">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>171</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="querentianjiabt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guanbibt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckAgain">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重新查询</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
